rules/manager.go

```diff
@@ -65,7 +65,7 @@ type Metrics struct {
 	groupRules          *prometheus.GaugeVec
 }
 
-// NewGroupMetrics makes a new Metrics and registers them with the provided registerer,
+// NewGroupMetrics makes a new instance of Metrics and registers them with the provided registerer,
```

Comment:
```suggestion
// NewGroupMetrics creates a new instance of Metrics and registers it with the provided registerer,
```

Reply:
@roidelapluie thank you for making it even better :)